src/server/master_server/master_metadata_service_impl.h

```diff
@@ -14,16 +17,22 @@ namespace service {
 class MasterMetadataServiceImpl final
     : public protos::grpc::MasterMetadataService::Service {
  public:
+  MasterMetadataServiceImpl(common::ConfigManager* config_manager) : 
+      config_manager_(config_manager) {}
+
   // Register a protocol client for talking to the lease and file service
   // of given |server_name| chunk server, using the gRPC |channel|.
   // If the client already exits, return false.
   bool RegisterChunkServerRpcClient(std::string server_name,
                                     std::shared_ptr<grpc::Channel> channel);
 
-protected:
+ protected:
   // Accessor to the MetadataManager instance
   server::MetadataManager* metadata_manager();
 
+  // Accessor to the ChunkServerManager instance
+  server::ChunkServerManager& chunk_server_manager();
```

Comment:
nit: ditto. You can get this directly too (its a singleton), instead of having methods. Reason for this?

Reply:
Writing "server::ChunkServerManager::GetInstance()" and "server::MetadataManager::GetInstance()" takes quite a bit of space, and I am basically using these methods as alias :) Furthermore, they are inlined (see cc file) so no extra overhead.